docs/testing/e2e.md

```diff
@@ -1,5 +1,7 @@
 **Running End-to-end Tests on Remote Clusters**
 
+**This document is for kubebuilder v1 only**
```

Comment:
I do not think that we should add this comment. 
if It has something that changed then, since the V1 is deprecated, IHMO we should update the doc to show how to do the same in the V2 instead of adding this note.

Reply:
I did what @mengqiy asked :) https://github.com/kubernetes-sigs/kubebuilder/pull/1208#discussion_r353362278

I can revert it, let me know

Reply:
Oh I see. So all is fine 👍